### DISCUSSION TITLE
Conditional text color or background color in cell

### DISCUSSION THREAD

**Discussion by Amtech281** [firecmsco/firecms#279] [Q&A]
Hi
We have two date fields in our collection. 
In the document list we want to highlight one date field either with red background color or with red text, whenever this rule is true: Date1>Date2.
example:
![grafik](https://user-images.githubusercontent.com/65586807/160685781-ce67fe76-5eda-4557-92e6-a5d6e75dbbd0.png)

Can this be implemented in Firecms?

**Comment by fgatti675** [comment]
Hi Andre, this can be achieved with a custom preview:
https://firecms.co/docs/entities/custom_previews

**Comment by Amtech281** [comment]
Hi Francesco, thanks for the fast answer!
In fact we are already using custom_previews, but it does not work fully as we would expect or like to have.

Our Rule is
IF UpdateDate>Version THEN make text in UpdateDate red. Otherwise black.

In the screenshot above:
IF UpdateDate changes and the rule for color is fulfilled, then text turns red (as in the screenshot)
IF now Version changes and the rule for color is not fulfilled anymore, and UpdateDate should become black again, this does not work.

Can this be solved somehow?

**Reply by fgatti675** [reply]
Ah, I see your use case. You would need to be able to access other values from the preview component.
That is not possible at the moment unfortunately

**Reply by Amtech281** [reply]
thanks for your feedback!
